utility/doc/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.4] - 2022-09-23
+
+- Consolidated `UtilActorType` in `utility` and `utility/types` to `typesUtil.ActorType`
+- Deprecated all code in `actor.go` and replaced with test helpers
+- Converted stake status to proto.enum (int32)
+- Added DISCUSS items around shared code and `StakeStatus`
```

Comment:
`// DISCUSS: *Design Decision* deprecating StakeStatus`

Is this the one you were referring to? If so, let's do it in a protocol hour or ad-hoc with the team this week.

Reply:
Yes